### PR TITLE
Feature: Multiple Prebid Servers

### PR DESCRIPTION
**Pull request by r-schweitzer**
## Type of change
- Feature

## Description of change
Allow for multiple prebid server configurations.
See original PR https://github.com/prebid/Prebid.js/pull/5209

## Other information
Issue: https://github.com/prebid/Prebid.js/issues/4980
Docs: https://github.com/prebid/prebid.github.io/pull/2512

### CODE REVIEW

**Comment by robertrmartinez** [comment]
@r-schweitzer Looks like some tests are still failing.

`Multiple S2S tests`

Otherwise I will take a look and give this a thorough review this week

File: modules/s2sTesting.js [hunk 1]
```diff
@@ -15,8 +15,11 @@ s2sTesting.globalRand = Math.random(); // if 10% of bidderA and 10% of bidderB s
 
 // load s2sConfig
 config.getConfig('s2sConfig', config => {
-  testing = config.s2sConfig && config.s2sConfig.testing;
-  s2sTesting.calculateBidSources(config.s2sConfig);
+  if (config && config.s2sConfig) {
+    const s2sConfigs = Array.isArray(config.s2sConfig) ? config.s2sConfig : [config.s2sConfig];
+    testing = s2sConfigs.some(i => i.testing);
```
Comment (robertrmartinez) [inline modules/s2sTesting.js:20]: so if one s2sconfig is true, all are?

Comment (r-schweitzer) [reply modules/s2sTesting.js:20]: Yes, this was from the original PR. Was this incorrect?

**Comment by robertrmartinez** [comment]
@r-schweitzer Hey, are we good to do a more formal / thorough review now?

Or are you still working on some stuff?

**Comment by r-schweitzer** [comment]
@robertrmartinez Good for you to start looking at again

**Review by robertrmartinez — APPROVED** [review]
This looks great to me.

Awesome job @r-schweitzer this was a dooooozy I am sure.

**Comment by robertrmartinez** [comment]
@Fawke This PR looks good to me. Whenever you have some time will you give it a second look?

I tested it locally and seemed to be working properly.

Thanks!

**Review by Fawke — APPROVED** [review]
LGTM. Thanks for implementing this feature.

**Pull request merged**